examples/k8s/otel-config.yaml

```diff
@@ -82,14 +82,6 @@ spec:
         volumeMounts:
         - name: otel-agent-config-vol
           mountPath: /conf
-        livenessProbe:
-          httpGet:
-            path: /
-            port: 13133 # Health Check extension default port.
-        readinessProbe:
-          httpGet:
-            path: /
-            port: 13133 # Health Check extension default port.
```

Comment:
Does the pod still run without health extension?

Reply:
In my observations it did. I assume it would just always report it coming up fine (marked "ready") even if it crashed/hung up. I think eventually it would be nice having such probes in place but in the state prior to the patch it would always be marked "pending" even if it came up fine, which can be a bit confusing for someone setting up their first example system like it was for me.

Reply:
Correct, it does. The health probes are used as a signal for Kubernetes to add the pod to the load balancer when it's ready, or remove it when it's not alive anymore. Not having the probes means that Kubernetes will add it to the pod as soon as the pod is started, and remove only when it really crashes.

Reply:
Removing this is as good as it gets right now without having health check port, right? Or would you prefer probing some other port?

Reply:
No, we should be removing this. This PR looks good to me.